src/main/java/com/sweater/controllers/NoteController.java

```diff
@@ -0,0 +1,54 @@
+package com.sweater.controllers;
+
+import com.sweater.entities.Notes;
+import com.sweater.repositories.NotesRepository;
+import com.sweater.services.NoteService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@Controller
+public class NoteController {
+    private NoteService noteService;
+
+    @Autowired
+    public NoteController(NoteService noteService) {
+        this.noteService = noteService;
+    }
+
+    @GetMapping("notes")
+    public ResponseEntity<List<Notes>> getAllNotes() {
```

Comment:
Use @ResponseBody instead ResponseEntity<List<Notes>>

Reply:
Or @RestController, 'cause @RestController is simple @Controller, but includes @ResponseBody